fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java

```diff
@@ -1841,7 +1841,7 @@ public PlanFragment visitPhysicalProject(PhysicalProject<? extends Plan> project
         List<Expr> projectionExprs = null;
         List<Expr> allProjectionExprs = Lists.newArrayList();
         List<Slot> slots = null;
-        if (project.hasMultiLayerProjection()) {
+        if (project.hasMultiLayerProjection() && !(inputFragment instanceof MultiCastPlanFragment)) {
```

Comment:
will we get wrong result if we just change logic in translator?
btw: add comment to explain why cannot do multi layer project

Reply:
when we add multiLayerProjection, we do not modify its original projects. That is why only translator need to be modified.